### PR TITLE
Update binance trade fetcher

### PR DESCRIPTION
**Pull request by halink0803**

### CODE REVIEW

File: lib/coreclient/client.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+	if err != nil {
+		return respBody, err
+	}
+	defer func() {
+		if cErr := resp.Body.Close(); cErr != nil {
+			logger.Errorw("Response body close error", "error", cErr.Error())
+		}
+	}()
+	switch resp.StatusCode {
+	case 200:
```
Comment (secmask) [inline lib/coreclient/client.go:95]: use http.StatusOK

File: lib/coreclient/client.go [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+		respBody, err = ioutil.ReadAll(resp.Body)
```
Comment (secmask) [inline lib/coreclient/client.go:96]: consider to move ReadAll out of switch block as we read body on all case

File: accounting/cmd/accounting-binance-frequent-trade-fetcher/main.go [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+		}
+
+		binanceFetcher := fetcher.NewFetcher(sugar, binanceClient, retryDelay, attempt, batchSize, binanceStorage, account.Name, marketDataClient)
+
+		if err := binanceFetcher.GetTradeHistory(fromIDs, tokenPairs, account.Name); err != nil {
+			return err
+		}
+	}
+
+	return binanceStorage.Close()
```
Comment (secmask) [inline accounting/cmd/accounting-binance-frequent-trade-fetcher/main.go:158]: should we use defer for Close here?

Comment (halink0803) [reply accounting/cmd/accounting-binance-frequent-trade-fetcher/main.go:158]: just realized that we already have defer func for storage close.

Comment (halink0803) [reply accounting/cmd/accounting-binance-frequent-trade-fetcher/main.go:158]: https://github.com/KyberNetwork/reserve-stats/blob/3babee9367424fca17cb51f65e8dc70087515d92/accounting/cmd/accounting-binance-frequent-trade-fetcher/main.go#L101

**Review by secmask — APPROVED** [review]

**Pull request merged**